### PR TITLE
Fix setuptools code snippet (dict, module path)

### PR DESCRIPTION
**Pull request by bittner**
I can't help but I find using a dictionary-style entry_points definition more pythonic.

### CODE REVIEW

File: docs/setuptools.rst [hunk 1]
```diff
@@ -126,14 +126,15 @@ These would be the modified contents of ``setup.py``::
 
     setup(
         name='yourpackage',
-        version='0.1',
+        version='0.1.0',
```
Comment (ThiefMaster) [inline docs/setuptools.rst]: please don't make unrelated changes

Comment (bittner) [reply docs/setuptools.rst]: Really? Quite common? Not with semver.

You're semi-right: It's a code snippet. I'm not changing the behavior of the application. I'm adapting the code snippet.

Comment (bittner) [reply docs/setuptools.rst]: BTW, I would also try to fix #1239 with this PR. I think it makes sense to do it in one go. Still a code snippet change.

Comment (ThiefMaster) [reply docs/setuptools.rst]: 0.1 and 0.1.0 have the same meaning even with semver, but i just checked the spec and they indeed expect you to always put x.y.z and not omit z if z==0...

Comment (bittner) [reply docs/setuptools.rst]: Thank you for verifying. :+1: So, it's okay that I leave it in there?

Comment (ThiefMaster) [reply docs/setuptools.rst]: Ideally it should be in a separate commit (it's a different change), but the change is trivial so I'll just go ahead and merge it as it is

**Review by ThiefMaster — APPROVED** [review]

**Comment by bittner** [comment]
Wow! That was fast! -- I've just noticed, there is another code snippet on the same page. I'll place another PR.

File: docs/setuptools.rst (hunk 1 above)
Comment (davidism) [reply docs/setuptools.rst]: Click has always been 2 numbers, not 3. Please revert this.

Comment (ThiefMaster) [reply docs/setuptools.rst]: @davidism it's not the click version but the example for people's own packages - otherwise i wouldn't have accepted it

Comment (davidism) [reply docs/setuptools.rst]: Yeah, I noticed right after, never mind.
